packages/scheduler/src/ScheduledTask.js

```diff
@@ -19,6 +19,7 @@ export default class ScheduledTask {
   }
 
   dispose () {
+    this.active = false
```

Comment:
This allows the scheduler to discard periodic tasks a bit earlier when they're canceled.